includes/class-wc-gateway-cardknox.php

```diff
@@ -420,10 +420,9 @@ public function payment_scripts()
         wp_enqueue_script('woocommerce_cardknox', plugins_url('assets/js/cardknox' . $suffix . '.js', WC_CARDKNOX_MAIN_FILE), array('jquery-payment'), filemtime(plugin_dir_path(dirname(__FILE__)) . 'assets/js/cardknox' . $suffix . '.js'), true);
         
 
-        $token_key = $this->token_key;
-        //$transaction_key = $this->transaction_key;
+        $encrypted_key = $this->token_key;
```

Comment:
Is this key encrypted? Does it make sense to leave the name as `$token_key`?

Reply:
Hello @mstein-ck 
No, this is not encrypted. But we have assigned $this->token_key in $tokey_key variable (as per the coding standard).
and this $tokeny_key variable is called into Cardknox key. Please see the screenshot.

![image](https://github.com/user-attachments/assets/fe8bda21-efaa-48ed-a5a7-e971f040c370)